templates/etc/systemd/system/docker.service.d/override.conf.j2

```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+
+[Service]
+ExecStart=
```

Comment:
systemd units can have multiple `ExecStart` commands. If you add a `ExecStart=/bin/foo` in an override it will additionaly start `/bin/foo` on when the unit is scheduled. With `ExecStart=` the "stack" of start commands gets emptied so the unit overridde can replace the ExecStart from the unit being overridden.